css/css-fonts/variations/font-variation-settings-parsing.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Testing the parsing of the font-variation-settings property</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
+    <script>
+        valueParserTests = [
+            { input: "'wght' 1000, '9 ~A' -45",           output: "\"9 ~A\" -45, \"wght\" 1000",    valid: true,  message: "Axis tag with valid non-letter ascii characters" },

```

Comment:
Would be nice to have names consistent between tests, e.g. input/output/valid->value/expectedComputedValue/isValid